app/assets/javascripts/student_profile/provided_by_educator_dropdown.js

```diff
@@ -47,9 +47,12 @@
     toggleOpenMenu: function () {
       $('.ProvidedByEducatorDropdown').autocomplete("search", "");
     },
-
+    closeMenu: function () {
+      $('.ProvidedByEducatorDropdown').autocomplete('close');
```

Comment:
I can use the react find DOM api, I wasn't sure how much I could refactor and decided to go for consistency. I'll push some refactors.